index.js

```diff
@@ -1,13 +1,39 @@
 class SortedList {
```

Comment:
remember to check for syntax errors. Since one parenthesis went missing, nothing works (even if the logic is good)